js/main.js

```diff
@@ -47,6 +48,11 @@ var adCapacity = adForm.querySelector('#capacity');
 
 var pinsFragment = renderPins();
 
+function getStylePropertyLikeNumber(element, property, pseudo) {
+  pseudo = typeof pseudo !== 'undefined' ?  pseudo : null;
```

Comment:
https://up.htmlacademy.ru/javascript/20/criteries#d18
```suggestion
  pseudo = pseudo ?  pseudo : null;
```